crates/bindings-csharp/Codegen.Tests/Sample.cs

```diff
@@ -1,5 +1,9 @@
-using System;
-using System.Collections.Generic;
+// Standard implicit usings.
+global using global::System;
+global using global::System.Collections.Generic;
+global using global::System.IO;
+global using global::System.Linq;
```

Comment:
Do all users need to add these to their code now?

Reply:
No, it's normally added implicitly by `<ImplicitUsings... >` directive which we have in the csproj template. It's just that the fest doesn't (can't) use csproj/msbuild infrastructure, so needs some extra manual work.